packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -110,19 +105,14 @@ export default {
             return this.ariaDescription ? this.uid : null;
         },
 
-        linkAttributes () {
-            return {
-                ...this.$attrs,
-                ...(this.isRouterLink ? {
-                    to: this.href
-                } : {
-                    href: this.href
-                })
-            };
-        },
-
         linkType () {
-            return this.isRouterLink ? 'router-link' : 'a';
+            if (this.$attrs.href) {
+                return 'a';
+            }
+            if (this.$attrs.to) {
+                return 'router-link';
+            }
+            return 'a';
```

Comment:
If you're defaulting to `a` you could shorten this down to use a ternary?

```suggestion
            return this.$attrs.to ? 'router-link' : 'a';
```

Reply:
Almost. I wanted to keep it in line with f-button, which prefers `href` over `to` if you pass both in.

https://github.com/justeat/fozzie-components/blob/master/packages/components/atoms/f-button/src/components/Button.vue#L124-L130

Reply:
Ah right, looks good to me then 😸